testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeAlterTableCompatibility.java

```diff
@@ -264,14 +265,14 @@ public void testOptimizeUnsupportedWriterVersion()
         onDelta().executeQuery(format("" +
                         "CREATE TABLE default.%s (col int) " +
                         "USING DELTA LOCATION 's3://%s/%s'" +
-                        "TBLPROPERTIES ('delta.minWriterVersion'='6')",
```

Comment:
why not keep 6 here?

from cmt msg

> Change delta.minWriterVersion to 7. Otherwise, Spark throws
delta.minWriterVersion needs to be one of 1, 2, 3, 4, 5, 7 error.

do you know why 6 is forbidden?

Reply:
I don't know. I asked the question in Delta Lake Slack 2 weeks ago, but there was no response. https://delta-users.slack.com/archives/CJ70UCSHM/p1686138318204249